examples/point_env.py

```diff
@@ -1,30 +1,40 @@
-from rllab.envs import Env
-from rllab.spaces import Box
-from rllab.envs import Step
+import gym
 import numpy as np
 
+from rllab.envs import Step
+from rllab.envs.env_spec import EnvSpec
+
 
-class PointEnv(Env):
+class PointEnv(gym.Env):
     @property
     def observation_space(self):
-        return Box(low=-np.inf, high=np.inf, shape=(2,))
+        return gym.spaces.Box(low=-np.inf, high=np.inf, shape=(2, ))
```

Comment:
Can rllab algorithms take gym.spaces.Box? Do the algorithms still run properly?

Reply:
yes